_run/kube/README.md

```diff
@@ -49,22 +41,23 @@ this example.
 |`OSEQ`|1|order sequence|
 |`PRICE`|10akash|price to bid|
 
-## Runbook
+# Runbook
 
 The following steps will bring up a network and allow for interacting
 with it.
 
-Running through the entire runbook requires four terminals.
+Running through the entire runbook requires three terminals.
 Each command is marked __t1__-__t3__ to indicate a suggested terminal number.
 
 If at any time you'd like to start over with a fresh chain, simply run:
 
-__t1__
+__t1 run__
 ```sh
-make clean kind-cluster-clean init
+make clean kind-cluster-clean 
+make init
```

Comment:
curious: is this for readability, consistency w/ later steps, or something else?

Reply:
If there is an error in the `kind-cluster-clean` step; is, it breaks, and then won't run the `init` component. So I split it into two lines so the init will run afterwards.